packages/apsara-ui/src/TableV2/VirtualisedTable.tsx

```diff
@@ -61,7 +71,7 @@ function VirtualisedTable({ columnsData, sortable = false, rowClick, dataFetchFu
         onSortingChange: setSorting,
         getCoreRowModel: getCoreRowModel(),
         getSortedRowModel: getSortedRowModel(),
-        debugTable: true,
+        debugTable: false,
```

Comment:
```suggestion
```

Reply:
Let's keep this one in disabled state, as it comes handy to debug table related issues